example/main.tf

```diff
@@ -2,10 +2,10 @@ terraform {
   required_version = "~> 0.14"
   required_providers {
     anaml = {
-      source  = "registry.anaml.io/anaml/anaml"
+      source = "nixpkgs/anaml"
     }
     anaml-operations = {
-      source  = "registry.anaml.io/anaml/anaml-operations"
+      source = "nixpkgs/anaml-operations"
```

Comment:
For testing?

Reply:
I've updated this to use `simple-machines/anaml-operations`, the same that would be used for real. The required Nix config to set this correctly was changed for the newer version of Terraform, to go along with the new plugin directory structure.